docs/en/seatunnel-engine/checkpoint-storage.md

```diff
@@ -32,17 +31,46 @@ seatunnel:
                   K1: V1 # plugin other configuration
                   K2: V2 # plugin other configuration   
 ```
+#### OSS
+Aliyun oss base on hdfs-file, so you can refer [hadoop oss docs](https://hadoop.apache.org/docs/stable/hadoop-aliyun/tools/hadoop-aliyun/index.html) to config oss.
```

Comment:
https://github.com/apache/incubator-seatunnel/blob/6e0e34b29eabfa100c00fac54d45651997d50953/docs/en/seatunnel-engine/checkpoint-storage.md#L10 you forget this.
and preferably in chronological order rather than random insertions

Reply:
Added OSS to these, all hdfs supported schema are OSS,S3,HDFS,LocalFile  now.